tools-local/tasks/mobile.tasks/WasmBundleTask/TimeZoneLoader.cs

```diff
@@ -14,70 +14,80 @@
 
 public class TimeZoneLoader : Task
 {
+    [Required]
     public string? InputDirectory { get; set; }
```

Comment:
The required ones shouldn't be nullable.

Reply:
They need to be nullable if they're uninitialized: 
`TimeZoneLoader.cs(18,19): error CS8618: Non-nullable property 'InputDirectory' is uninitialized. Consider declaring the property as nullable. [/Users/tammyqiu/Desktop/Desktop/runtime/tools-local/tasks/mobile.tasks/WasmBundleTask/WasmBundleTask.csproj]`